src/bio/bio_xstream.c

```diff
@@ -972,9 +973,32 @@ init_bio_bdevs(struct bio_xs_context *ctxt)
 
 		rc = create_bio_bdev(ctxt, spdk_bdev_get_name(bdev), NULL);
 		if (rc)
-			break;
+			return rc;
 	}
-	return rc;
+
+	for (bdev = spdk_bdev_first(); bdev != NULL; bdev = spdk_bdev_next(bdev)) {
+		if (nvme_glb.bd_bdev_class != get_bdev_type(bdev))
+			continue;
+
+		d_bdev = lookup_dev_by_name(spdk_bdev_get_name(bdev));
```

Comment:
why not directly call this lookup_dev_by_name() & bio_led_mange() in above loop?

Reply:
The led reset called through bio_led_manage() fails when checking all the devices for faulty state if the blobstores are not created for all bdevs. check_faulty() calls bio_dev_list() which requires the above loop to have completed. Make sense?

Reply:
I see, thanks.